File: migrate-imports.py
import os

def migrate_imports(directory):
    modified = 0
    for root, _, files in os.walk(directory):
        for file in files:
            if file.endswith('.ts') or file.endswith('.tsx'):
                filepath = os.path.join(root, file)
                with open(filepath, 'r') as f:
                    content = f.read()
                
                new_content = content.replace('@/lib/firebase/db', '@/lib/mongo/db')
                
                if new_content != content:
                    with open(filepath, 'w') as f:
                        f.write(new_content)
                    print(f"Updated: {filepath}")
                    modified += 1
    
    print(f"Total files modified: {modified}")

if __name__ == "__main__":
    migrate_imports('src')

File: migrate-imports.js
const fs = require('fs');
const path = require('path');

function walkDir(dir, callback) {
  fs.readdirSync(dir).forEach(f => {
    let dirPath = path.join(dir, f);
    let isDirectory = fs.statSync(dirPath).isDirectory();
    isDirectory ? walkDir(dirPath, callback) : callback(path.join(dir, f));
  });
}

let modifiedCount = 0;

walkDir('./src', function(filePath) {
  if (filePath.endsWith('.ts') || filePath.endsWith('.tsx')) {
    const originalContent = fs.readFileSync(filePath, 'utf8');
    const newContent = originalContent.replace(/@\/lib\/firebase\/db/g, '@/lib/mongo/db');
    if (originalContent !== newContent) {
      fs.writeFileSync(filePath, newContent, 'utf8');
      modifiedCount++;
      console.log(`Updated: ${filePath}`);
    }
  }
});

console.log(`Total files modified: ${modifiedCount}`);
